apps/ledger-live-mobile/src/families/evm/SelectFeesStrategy.tsx

```diff
@@ -0,0 +1,255 @@
+import { getEstimatedFees } from "@ledgerhq/coin-evm/logic";
+import { getTypedTransaction } from "@ledgerhq/coin-evm/transaction";
+import type { GasOptions, Transaction } from "@ledgerhq/coin-evm/types";
+import { getFeesCurrency, getFeesUnit, getMainAccount } from "@ledgerhq/live-common/account/index";
+import type { Account, AccountLike } from "@ledgerhq/types-live";
+import { useTheme } from "@react-navigation/native";
+import React, { useCallback, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  FlatList,
+  ListRenderItemInfo,
+  SafeAreaView,
+  StyleSheet,
+  TouchableOpacity,
+  TouchableOpacityProps,
+  View,
+} from "react-native";
+import { useAnalytics } from "../../analytics";
+import CounterValue from "../../components/CounterValue";
+import CurrencyUnitValue from "../../components/CurrencyUnitValue";
+import LText from "../../components/LText";
+import NetworkFeeInfo from "../../components/NetworkFeeInfo";
+import QueuedDrawer from "../../components/QueuedDrawer";
+import SectionSeparator from "../../components/SectionSeparator";
+import Info from "../../icons/Info";
+import TachometerFast from "../../icons/TachometerFast";
+import TachometerMedium from "../../icons/TachometerMedium";
+import TachometerSlow from "../../icons/TachometerSlow";
+import SummaryRow from "../../screens/SendFunds/SummaryRow";
+import { sharedSwapTracking } from "../../screens/Swap/utils";
+import BigNumber from "bignumber.js";
+import invariant from "invariant";
+
+type Strategy = keyof GasOptions;
+type StrategyWithCustom = Strategy | "custom";
```

Comment:
Seems like we use `keyof GasOptions` a lot
I would put this type in `coin-evm` as well